src/components/Login/LoginContainer.tsx

```diff
@@ -133,12 +124,17 @@ export const InitialisationContainer: FunctionComponent<NavigationProps> = ({
     });
   };
 
+  const setState = useState()[1];
   const handleRequestOTP = async (
     fullNumber = mobileNumber
   ): Promise<boolean> => {
     try {
       if (!(await getResendConfirmationIfNeeded())) return false;
-      const response = await requestOTP(fullNumber, codeKey, endpointTemp);
+      const response = await requestOTP(
+        fullNumber,
+        tempAuthCredentials?.operatorToken ?? "",
```

Comment:
May be a noob question, but why is it possible for `operatorToken` and `endpoint` to be empty? Is this for dev environment?